Model/Push/DefaultProcessor.php

```diff
@@ -720,6 +720,13 @@ public function processSucceededPush(string $newStatus, string $message): bool
 
         $this->processSucceededPushAuthorization();
 
+        if ($this->configAccount->getInvoiceHandling() == InvoiceHandlingOptions::SHIPMENT) {
+            $paymentDetails['state'] = Order::STATE_NEW;
+            $paymentDetails['newStatus'] = $this->configAccount->getOrderStatusPending();
```

Comment:
why do you use array here, since you send the individual values to the orderRequestService?

Reply:
I used that array here app/code/Buckaroo/Magento2/Model/Push/DefaultProcessor.php:711 
$saveInvoice = $this->invoiceShouldBeSaved($paymentDetails);

$paymentDetails can be changed in invoiceShouldBeSaved